src/pages/Pool/index.tsx

```diff
@@ -121,17 +143,11 @@ export default function Pool() {
                     Add Liquidity
                   </Text>
                 </ResponsiveButtonPrimary>
-                {/* <ResponsiveButtonPrimary
-                  id="join-pool-button"
-                  as={Link}
-                  padding="5px 8px"
-                  borderRadius="12px"
-                  to={`/remove/${DFI[chainId!].address}/ETH`}
-                >
+                <ResponsiveButtonPrimary padding="5px 8px" borderRadius="12px" onClick={() => addTokenFunction()}>
```

Comment:
Will you be able to hide the button when chainId is not set to 1, or Web3 wallet not connected?

Reply:
It will make sense to have this for Account as we already using infura for pools Apr. This might result in two different chain id. For example, chain ID we have is 1 but the user might have a different chain selected. Account will only work for mainet and goerli.

Reply:
Very nice, thank you! Exactly my thoughts too!